build.sbt

```diff
@@ -94,35 +92,41 @@ def releaseSettings: Seq[Setting[_]] = Seq(
   }
 )
 
+
+def latestScalafmt = "1.0.0-RC4"
+commands += Command.args("scalafmt", "Run scalafmt cli.") {
+  case (state, args) =>
+    val Right(scalafmt) =
+      org.scalafmt.bootstrap.ScalafmtBootstrap.fromVersion(latestScalafmt)
+    scalafmt.main("--non-interactive" +: args.toArray)
+    state
+}
+
+
 def runtimeLibCommon: Seq[Setting[_]] = common ++ Seq(
   crossScalaVersions := Seq(Dependencies.ScalaVersion),
   scalaVersion := crossScalaVersions.value.head,
   crossVersion := CrossVersion.binary,
   crossPaths := false,
-
   Dependencies.validateDependenciesSetting,
   Dependencies.dependencyWhitelistSetting,
-
   // compile options
-  scalacOptions in Compile ++= Seq("-encoding", "UTF-8", "-target:jvm-1.8", "-feature", "-unchecked", "-Xlog-reflective-calls", "-Xlint", "-deprecation"),
-
+  scalacOptions in Compile ++= Seq("-encoding",
```

Comment:
In cases like this, I like a newline after the opening `(` and before the closing `)` (as with `javacOptions` above) so it doesn't push everything so far to the right.